chore: updated editors page with corrections and a new jetbrains licensing page (#768)
coder/docs: guides/troubleshooting/activate-jetbrains-licensing.md

---
title: "Activate JetBrains license in a browser"
description:
  Learn how to activate a license when running a JetBrains IDE in a browser.
---

JetBrains requires a valid license to evaluate or use a JetBrains IDE. When
running a JetBrains IDE in a browser, you need to perform the following steps.

1. [Click the JetBrains IDE icon](activate-jetbrains-licensing.md#launch-the-jetbrains-ide-in-a-browser)
   in your Coder workspace

> 1. [Click the Log In to JetBrains Account button](activate-jetbrains-licensing.md#log-into-your-jetbrains-account)

Reviewer: ```suggestion
1. [Click the "Log In to JetBrains Account" button](activate-jetbrains-licensing.md#log-into-your-jetbrains-account)
```

> 1. [Click the Troubles link](activate-jetbrains-licensing.md#click-the-troubles-link)
>    since you are already in a browser

Reviewer: ```suggestion
1. Since your IDE is in the web browser, click the ["Troubles" link](activate-jetbrains-licensing.md#click-the-troubles-link) to use an alternate login method 
```

> 1. [Click the copy link](activate-jetbrains-licensing.md#click-the-copy-link),

Reviewer: ```suggestion
1. [Click the "copy" link](activate-jetbrains-licensing.md#click-the-copy-link),
```

   open a new browser window and paste the link
1. [Log into JetBrains with your valid JetBrains credentials](activate-jetbrains-licensing.md#login-with-your-jetbrains-account)
   to get an IDE authentication token

> 1. [Click Copy token](activate-jetbrains-licensing.md#click-copy-token) to copy

Reviewer: ```suggestion
1. [Click "Copy token"](activate-jetbrains-licensing.md#click-copy-token) to copy
```
